Assets/Scripts/Ship/Ship.cs

```diff
@@ -4,22 +4,38 @@
 public class Ship : MonoBehaviour
 {
     public int shipHealth = 3;
-    public float delayTime = 0.01f;
-   
+    public Sprite healthBar3;
+    public Sprite healthBar2;
+    public Sprite healthBar1;
+    public Sprite healthBar0;
```

Comment:
Let's put these into an array. Will look much better. When you have this much repetition of variables then you've likely done something wrong.